packages/core/src/models/promptVariants.ts

```diff
@@ -24,25 +24,26 @@ export class PromptVariants {
     this.variant = foundVariant;
   }
 
-  async setCurrent() {
-    const template = this.format();
-    const hash = Md5.hashStr(template);
+  async setCurrent(variant: string, raw = false) {
+    const template = this.format(raw);
+    const { slug } = this.definition;
+    const hash = `${slug}-${variant}-${Md5.hashStr(template)}`;
+    console.log("*** setCurrent ***");
+    console.log("Hash", hash);
+    console.log("Slug", slug);
+    console.log("Variant", variant);
+    console.log("-----");
```

Comment:
Are you wanting to keep these in?

Reply:
Removing this but it's useful to know if it is inserting when you run the script. I'll add a line only if it inserts